pkg/mcs/resource_manager/server/server.go

```diff
@@ -275,6 +279,9 @@ func (s *Server) startServer() error {
 
 	// Server has started.
 	atomic.StoreInt64(&s.isServing, 1)
+	// register to etcd
+	s.serviceRegister = discovery.NewServiceRegister(s.ctx, s.etcdClient, "resource_manager", s.cfg.ListenAddr, s.cfg.ListenAddr, 3)
```

Comment:
3？

Reply:
3 seconds

Reply:
how about refactor the function's signature for the ttl unit, and make it more clear.

Reply:
If more clear, it will be better.

Reply:
How about using a constant? like leaseInSeconds? I just keep consistent with etcd.